source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -147,6 +147,37 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
       # systemctl start rpcbind 
       # systemctl start nfs-server
 
+#### Synology DSM 7
+Synology DSM 7 adds / extends Access Control Lists (ACLs) on all shared folders, and removes `groupadd` and `useradd`...  
+To allow ovirt nodes to connect via NFS the ACL permissions and /etc/exports file need to be updated manually (*take backups before editing!*).
+
+1. Create the shared folder required as usual and allow you oVirt hosts IP's in the NFS permissions section
+
+2. Login to you Synology NAS via SSH (search the web for 'synology enable ssh access') as root or sudo to root once logged in
+
+3. Carefully add kvm (id 36) to /etc/group and vdsm (id 36) to /etc/passwd and /etc/shadow, if you need an encrypted password the python command below will create one
+
+        # python3 -c 'import crypt,getpass;pw=getpass.getpass();print(crypt.crypt(pw) if (pw==getpass.getpass("Confirm: ")) else exit())'
```

Comment:
Is this really the best way to add a user/group in Synology NAS?

Reply:
DSM 7 removes user / group add commands from the shell...  The built-in user management in the UI and the Synology replacement user add command (synouser I think) offer zero control of UID / GID numbers.

Actually I'd really just recommend avoiding Synology like the plague.